src/Models/LagrangianParticleTracking/drogued_dynamics.jl

```diff
@@ -0,0 +1,50 @@
+"""
+    DroguedDynamics(depths)
+
+`DroguedDynamics` goes in the `dynamics` slot of `LagrangianParticles` 
+and modifies their behaviour to mimic the behaviour of bouys which are 
+drogued at `depths`. The particles remain at the their `z` position
+so the "measurment depth can be set", and then are advected in the `x` and `y`
+directions according to the velocity field at `depths`.
+
+`depths` should be an array of length `length(particles)`.
```

Comment:
sorry for the last minute suggestion but an example of building this for the docstring would be useful I think